cmd/katapult/main.go

```diff
@@ -33,6 +36,9 @@ func run() error {
 
 	rootFlags := rootCmd.PersistentFlags()
 
+	rootFlags.StringVarP(&outputFlag, "output", "o", "", "output type (yaml, json, text)")
+	rootFlags.StringVarP(&templateFlag, "template", "t", "", "defines the output template for text")
```

Comment:
Can I suggest we rename the `--template` flag to `--format` to keep it more consistent with Docker and other CLI tools with similar options?

Also I don't think we need a short variant for this flag, (specially if we call it `--format` as it's shorter), cause it's probably not likely to be used that often, and both `-t` and `-f` are quite common flags I can see us wanting to use in a few different sub-commands.